### ISSUE TITLE
Add additional notes to releases

### ISSUE THREAD

**Issue by kltm** [geneontology/go-site#1441]
We need to add some extra information to recent releases. For starters:

- [ ] sgd qualifier drop
- [ ] sgd qualifier spacing

**Comment by kltm** [comment]
Opened a PR here to add to: https://github.com/geneontology/go-site/pull/1442

**Comment by kltm** [comment]
Tagging @pgaudet

**Comment by pgaudet** [comment]
You want to list problems as tickets ? or action items? 

My observations are here: https://docs.google.com/document/d/1IMA54ycbHZxkFbIAyjRvzm_ybAaAj2V8q-Q670mE07U/edit#

I try to open tickets when I see something actionable.  For exmaple contributes to (with incorrect case) is mentioned here: 
https://github.com/geneontology/go-site/issues/1426

I didn't realize Rule 1 was a filter, I thought some aspects should be REPAIR. 


Let me know what you want to do next. 

Thanks, Pascale

**Comment by kltm** [comment]
As release notes. Take a look at the PR to get an idea, or we can talk later on.

**Comment by pgaudet** [comment]
Looks OK to me.

**Comment by kltm** [comment]
Right, but what other oddities have we discovered after the fact or let out the door over the last year? For example, what we did with SGN (I'd have to look back to remember if/what).

**Comment by kltm** [comment]
Taking the PR for now. Can add more as we go.
